scripts/buildocaml.sh

```diff
@@ -2,25 +2,30 @@
 set -e
 
 
-export OCAMLPARAM='_,bin-annot=1' 
+export OCAMLPARAM='_,bin-annot=1'
 export OCAMLRUNPARAM=b
 
 if [ $BS_TRAVIS_CI ]
-then 
+then
     git submodule update --init --recursive
     OCAML=ocaml
-else 
+else
     OCAML=ocaml_src
     if [  -f "ocaml.tar.gz" ]
-    then 
+    then
         echo "ocaml.tar.gz already exists"
         rm -rf $OCAML
         mkdir -p $OCAML
         tar -zxvf ./ocaml.tar.gz -C $OCAML
     else
         echo "creating ocaml from github"
-        git clone -b master --depth 1 https://github.com/bloomberg/ocaml $OCAML
+        if [ -d "$OCAML" ]
+        then
```

Comment:
are you sure this would fix your problem? Such codepath, creating from github would only happen for head branch, for npm published branch, it should not